### DISCUSSION TITLE
Is it ok to print a copy for personal use?

### DISCUSSION THREAD

**Discussion by SpencerGreene** [calcmogul/controls-engineering-in-frc#18] [Q&A] [answered]
Hi Tyler - this is an amazing resource and I want to echo all those others who have thanked you for it, times a thousand.  I was looking for a printed version and couldn't find one - but there are sites that will print and bind a one-off, so I wanted to check with you if it's ok for me to do that (one copy just for personal use)?  thanks! Spencer

**Comment by calcmogul** [comment, marked as answer]
Glad you like it. :)

License tl;dr is you may print a copy for any purpose without asking as long as credit is given and the license is linked (both of these are already done in the PDF). I appreciate the courtesy though. I added a section to the readme to make that clearer: https://github.com/calcmogul/controls-engineering-in-frc#printed-copies. Make sure you use the prepress copy instead of the normal one for professional printing.
